src/opensubtitlescom/config.py

```diff
@@ -0,0 +1,55 @@
+"""
+This file is part of Opensubtitles API wrapper.
+
+Opensubtitles API is free software: you can redistribute it and/or modify
+it under the terms of the MIT License as published by the Massachusetts
+Institute of Technology.
+
+For full details, please see the LICENSE file located in the root
+directory of this project.
+"""
+import os
+import json
+from typing import Optional
+from pathlib import Path
+
+
+DEFAULT_CONFIG_PATH = "~/.config/opensubtitlescom/cli.json"
```

Comment:
I don't see any way for a person using this library as a library to override this, which means any app which uses this library will have a `cli.json` created?

Reply:
This seems to be mixing two different approaches to config management, I think it'd make sense to pick one of:

- the library should have a single config, no matter which app is using the library (in which case I'd name it something like `config.json` rather than `cli.json`)

or

- each app should manage its own config (in which case rather than having a `Config` object in the library folder, I'd have the library just accept username / password etc as function parameters like it does now, and we let the application (in this case the CLI) handle loading and saving config variables)

Reply:
You're right, the name should be changed to config.json, as it is currently misleading. 
The usage remains the same; when used as a library, you can provide your own configuration. 
The JSON config file serves both the CLI and functions as the default for the library.